statusHandler/view/termuic/termuiRenders/widgetsRender.go

```diff
@@ -250,12 +250,12 @@ func (wr *WidgetsRender) prepareBlockInfo() {
 	rows[3] = []string{fmt.Sprintf("Current block hash : %s", currentBlockHash)}
 
 	crossCheckBlockHeight := wr.presenter.GetCrossCheckBlockHeight()
-	rows[4] = []string{fmt.Sprintf("Cross check block height: %s", crossCheckBlockHeight)}
+	rows[4] = []string{fmt.Sprintf("Block height: %s", crossCheckBlockHeight)}
```

Comment:
Might have something like "Cross check / final nonce: %d / %d" as it is more concise.

Reply:
Done.